lib/bolt/module_installer/specs/git_spec.rb

```diff
@@ -89,6 +88,15 @@ def to_hash
           }
         end
 
+        # Returns an error message that the provided repo is not a git repo or
+        # is private.
+        #
+        private def invalid_git_msg(repo_name)
+          "#{repo_name} is not a git repository or is private. If the repository is private, "\
+            "configure Bolt to not resolve dependencies for this module. See "\
+            "https://pup.pt/no-resolve for more information."
```

Comment:
This error will show for public non-GitHub repos as well and might be misleading since it implies that only private repos can use this approach. Maybe updating the message to say "is not a public GitHub repository" instead of "is not a git repository or is private"?

Reply:
Hm, I can see that. Gave it another shot!